app/models/reports/requests_by_type_report.rb

```diff
@@ -9,13 +9,6 @@ def description
       'A basic report that collects all requests by type'
     end
 
-    # this is actually not great....but, maybe we can figure this out..
-    # currently, this loads all the requests into memory. idealls, we can
-    # have this lazy load during iteration...
-    def query
-      [StaffRequest.enum_for(:find_each), ContractorRequest.enum_for(:find_each), LaborRequest.enum_for(:find_each)]
-    end
-
     def formats
       %w( xlsx )
     end
```

Comment:
GitHub won't let me leave a comment on the actual line, but in the "worksheets" comment, it should probably be "worksheet title".